advisor/src/test/assets/response_vulnerability_template.json

```diff
@@ -0,0 +1,49 @@
+{
+  "url": "http://localhost:8000/api/vulnerabilities/<<id>>/",
+  "resolved_packages": [
+    {
+      "url": "http://localhost:8000/api/packages/54923/",
+      "purl": "pkg:deb/debian/mimetex@1.50-1.1?distro=sid"
+    },
+    {
+      "url": "http://localhost:8000/api/packages/54924/",
+      "purl": "pkg:deb/debian/mimetex@1.50-1.1?distro=bullseye"
+    },
+    {
+      "url": "http://localhost:8000/api/packages/54925/",
+      "purl": "pkg:deb/debian/mimetex@1.76-1?distro=sid"
+    },
+    {
+      "url": "http://localhost:8000/api/packages/54928/",
+      "purl": "pkg:deb/debian/mimetex@1.76-1?distro=buster"
+    },
+    {
+      "url": "http://localhost:8000/api/packages/54929/",
+      "purl": "pkg:deb/debian/mimetex@1.50-1.1?distro=buster"
+    },
+    {
+      "url": "http://localhost:8000/api/packages/54927/",
+      "purl": "pkg:deb/debian/mimetex@1.76-1?distro=bullseye"
+    }
+  ],
+  "unresolved_packages": [
+    {
+      "url": "http://localhost:8000/api/packages/54922/",
+      "purl": "pkg:deb/debian/mimetex@1.74-1?distro=stretch"
+    },
+    {
+      "url": "http://localhost:8000/api/packages/54926/",
+      "purl": "pkg:deb/debian/mimetex@1.50-1.1?distro=stretch"
+    }
+  ],
+  "references": [
+    {
+      "source": "some_source",
+      "reference_id": "some_reference",
+      "url": "someUrl"
+    }
+  ],
+  "cve_id": "<<cve>>",
+  "summary": "some summary",
+  "cvss": <<score>>
```

Comment:
The GitHub rendering looks like you should put double-quotes here (like you do two lines above).

Reply:
As the value is numeric, I cannot use double-quotes. Maybe I can just rename the file to response_vulnerability.template because it is not a valid JSON file.

Reply:
Good idea. I'd propose `advisor/src/test/assets/response_vulnerability.json.template` then.

Reply:
Renamed the file.